### ISSUE TITLE
QWAT CH - Applied on german translation

### ISSUE THREAD

**Issue by ponceta** [qwat/QWAT#308]
One of the brakes to the use of QWAT in Switzerland is its missing translation to german.

The users should be provided with

- [ ] Translatable qgs demo project -> like QGEP
- [ ] availability of value lists in german
- [ ] translators and translation of the documentation -> 
https://www.transifex.com/qwat/qwat-doc/dashboard/
- [ ] SIA 405 interlis export (https://github.com/qwat/QWAT/issues/307)

![image](https://user-images.githubusercontent.com/8614223/94889622-1edaa200-047d-11eb-95aa-647ec9d856d2.png)

**Comment by afink70** [comment]
Hi, how could I help you?

**Comment by ponceta** [comment]
> Hi, how can I help you?

Please feel free to check the documentation about contributing 👍 

https://qwat.github.io/docs/master/en/html/contributor-guide/index.html#translating-qwat

Available if you encounter any trouble on how to contribute.

**Comment by lbartoletti** [comment]
@ponceta BTW, the english version still contains french sentences.

**Comment by ponceta** [comment]
> @ponceta BTW, the english version still contains french sentences.

@lbartoletti Yes, thank you, I add this to the list, there's an existing issue for that!
